tests/unit_tests/external_interaction/test_write_rotation_nexus.py

```diff
@@ -71,6 +75,152 @@ def plan():
     return plan()
 
 
+def dectris_device_mapping(meta_filename: str):
```

Comment:
Perhaps when we get more sample files with other detectors we could factor this out and drive this from the test data files